src/js/components/Form/__tests__/Form-test.js

```diff
@@ -581,4 +587,80 @@ describe('Form', () => {
     );
     expect(container.firstChild).toMatchSnapshot();
   });
+
+  test('uncontrolled without name', () => {
+    const onSubmit = jest.fn();
+    const { getByPlaceholderText, getByText, container } = render(
+      <Form onSubmit={onSubmit}>
+        <FormField>
+          <TextInput placeholder="test input" />
+        </FormField>
+        <Button type="submit" primary label="Submit" />
+      </Form>,
+    );
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.change(getByPlaceholderText('test input'), {
+      target: { value: 'v' },
+    });
+    expect(container.firstChild).toMatchSnapshot();
+    fireEvent.click(getByText('Submit'));
+    expect(onSubmit).toBeCalled();
+    expect(container.firstChild).toMatchSnapshot();
+  });
+
+  test('controlled reset', () => {
+    const onSubmit = jest.fn();
+    const onReset = jest.fn();
+    const Test = () => {
+      const [value, setValue] = React.useState({ test: '' });
+      const onChange = React.useCallback(nextValue => setValue(nextValue), []);
+      return (
+        <Grommet>
+          <Form
+            onReset={onReset}
+            onChange={onChange}
+            value={value}
+            onSubmit={onSubmit}
+          >
+            <FormField name="test" required placeholder="test input" />
+            <Button type="reset" primary label="Reset" />
+          </Form>
+        </Grommet>
+      );
+    };
+    const { getByPlaceholderText, getByText, queryByText } = render(
+      <Grommet>
```

Comment:
Here you are wrapping `<Test/>` inside of a `<Grommet/>` when it `<Test/>` is already wrapped inside of a `<Grommet/>` component

Reply:
Updated so it is no longer double wrapped in <Grommet>